### PR TITLE
Re-enable Windows tutorial + copyedits and refactoring

### PR DESCRIPTION
**Pull request by davidwrede**
- Adding back in the Windows version of the tutorial after testing the changes out with 0.29.1.

- Did a bit of minor refactoring and copyedits

Signed-off-by: David Wrede <dwrede@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: www/source/tutorials/sample-app/windows/build-package.html.slim [hunk 1]
```diff
@@ -9,7 +9,7 @@ section.sample-app
 
     p The studio is an isolated environment used to build and test Habitat packages. For Windows platforms, there are two versions of the studio. One uses a Linux-based Docker container to create a #{link_to 'chrooted', 'https://en.wikipedia.org/wiki/Chroot'} shell environment and the other is a native Windows environment. Because this tutorial involves using some functionality that is currently only available for Linux-based packages, we will use the Linux-based studio that resides in a Docker container.
 
-    p The studio itself is a Habitat package, and all of the dependent packages for the studio will be downloaded, unpacked, and installed. The studio will also import any secret origin keys created by <code>hab setup</code>. When running macOS or Windows on your host machine, the studio runs inside a Docker container as root.
+    p The studio itself is a Habitat package, and all of the dependent packages for the studio will be downloaded, unpacked, and installed. The studio will also import any secret origin keys created by <code>hab setup</code>. When Windows on your host machine, the studio runs inside a Docker container as root.
```
Comment (mwrock) [inline www/source/tutorials/sample-app/windows/build-package.html.slim]: `When Windows on your host machine` => `When running Windows on your host machine`

**Comment by reset** [comment]
@thesentinels approve

tenor-12321254

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/264577141).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/264577141).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**